care/abdm/api/viewsets/health_facility.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.mixins import (
+    CreateModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+)
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from care.abdm.api.serializers.health_facility import HealthFacilitySerializer
+from care.abdm.models import HealthFacility
+from care.abdm.utils.api_call import Bridge
+from care.utils.queryset.facility import get_facility_queryset
+
+
+class HealthFacilityViewSet(
+    GenericViewSet,
+    CreateModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+):
+    serializer_class = HealthFacilitySerializer
+    model = HealthFacility
+    queryset = HealthFacility.objects.all()
```

Comment:
List and Retrieve have no authZ, was that intended?

Reply:
no, intended to add it but forgot later

added it